display_list/display_list.cc

```diff
@@ -59,13 +59,16 @@ uint32_t DisplayList::next_unique_id() {
 
 class Culler {
  public:
+  virtual ~Culler() = default;
   virtual bool init(DispatchContext& context) = 0;
   virtual void update(DispatchContext& context) = 0;
 };
-class NopCuller : public Culler {
+class NopCuller final : public Culler {
```

Comment:
Was adding the final necessary to address the check or was it just a drive by change?

Reply:
Some of the suggested fixes I saw said that final sub-classes would also avoid the error, so I did both.

Reply:
The compiler should have been able to see that these classes were effectively final since they are defined in a .cc file and that they don't need virtual destructors because they are constructed and destructed locally within a single method, but I figured I would make everything as explicit as possible.